localisation/00_zuwc_united_localisation.csv

```diff
@@ -2848,6 +2848,14 @@ GetMonstrousUndeadNameDesc_Abomination;a horrific creation that is sewn together
 GetMonstrousUndeadNameDesc_WickerGolem;a hulking creation that is crafted from dead objects such as flesh, fur and bone and animated through an ancient death magic;;eine grobe Sch�pfung, die aus toten Materialien wie Fleisch, Fell und Knochen hergestellt und durch eine alte Todesmagie belebt wird;;;;;;;;;;;x
```

Comment:
"dead objects" should be replaced with "materials" as this would make far more sense.

Reply:
Not related to this branch, so it's done in 9e09c8bd1a56f45b2ff6743fd28e558e33b7f65b.